config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/ABeginnersGuidet-NGG2rHQ4T4SInwXT8XTGSg==.json

```
@@ -8,7 +8,7 @@
   "properties:10": {
     "betterquesting:10": {
       "autoClaim:1": 0,
-      "desc:8": "If you\u0027ve built the Source Chamber already, you may have noticed that particles aren\u0027t produced as items, like GT++ ones are. \n\nInstead they\u0027re transmitted through Beamline Pipes from multi to multi, similar to real life.\n\nEach stream of particles transmitted, or \"beam\", has four properties:\n- §\n§\n§6Particle§r: the type of particle carried by the beam\n- §4Energy§r: the amount of energy the particles have\n§r- §9Focus§r: out of 100, how focused the beam is\n§r- §dAmount§r: a measure of the number of particles in the beam\n\nNote that you can see info about a multi\u0027s input and output beams by using a Tricorder on the controller.\n\nFor the recipes you\u0027ll be interested in, i.e. the ones that actually produce items, there\u0027ll be a threshold for allowed input beam energy, a minimum focus value, and an amount value which determines recipe time (more on that in the Target Chamber quest). \nTake a look some TC recipes for an example of this.\n\n\nHave a photon. To hang on your wall.",
+      "desc:8": "If you\u0027ve built the Source Chamber already, you may have noticed that particles aren\u0027t produced as items, like GT++ ones are.\n\nInstead they\u0027re transmitted through Beamline Pipes from multi to multi, similar to real life.\n\nEach stream of particles transmitted, or \"beam\", has four properties:\n- §\n§\n§6Particle§r: the type of particle carried by the beam\n- §4Energy§r: the amount of energy the particles have\n§r- §9Focus§r: out of 100, how focused the beam is\n§r- §dAmount§r: a measure of the number of particles in the beam\n\nNote that you can see info about a multis input and output beams by using a Tricorder on the controller.\n\nFor the recipes you\u0027ll be interested in, i.e. the ones that actually produce items, there\u0027ll be a threshold for allowed input beam energy, a minimum focus value, and an amount value which determines recipe time (more on that in the Target Chamber quest).\nTake a look some TC recipes for an example of this.\n\n\nHave a photon. To hang on your wall.",
```

I restored the apostrophe (`multi's`) since this is possessive

```suggestion
      "desc:8": "If you\u0027ve built the Source Chamber already, you may have noticed that particles aren\u0027t produced as items, like GT++ ones are.\n\nInstead they\u0027re transmitted through Beamline Pipes from multi to multi, similar to real life.\n\nEach stream of particles transmitted, or \"beam\", has four properties:\n- §\n§\n§6Particle§r: the type of particle carried by the beam\n- §4Energy§r: the amount of energy the particles have\n§r- §9Focus§r: out of 100, how focused the beam is\n§r- §dAmount§r: a measure of the number of particles in the beam\n\nNote that you can see info about a multi\u0027s input and output beams by using a Tricorder on the controller.\n\nFor the recipes you\u0027ll be interested in, i.e. the ones that actually produce items, there\u0027ll be a threshold for allowed input beam energy, a minimum focus value, and an amount value which determines recipe time (more on that in the Target Chamber quest).\nTake a look some TC recipes for an example of this.\n\n\nHave a photon. To hang on your wall.",
```